examples/bri-3/src/bri/identity/bpiSubjects/api/dtos/request/updateBpiSubject.dto.ts

```diff
@@ -1,4 +1,5 @@
 export interface UpdateBpiSubjectDto {
+    id: string;
```

Comment:
i dont think id should be part of update dto but rather part of api route

Reply:
You are completely right. Fixed.